.github/workflows/test-plugin.yml

```diff
@@ -57,26 +44,25 @@ jobs:
         echo "PATH=${HOME}/.local/bin:${PATH}" >> "${GITHUB_ENV}"
 
     - name: Check pre-commit
+      working-directory: triton_shared/triton
       run: |
-        cd triton
-        python3 -m pip install --upgrade pre-commit
-        python3 -m pre_commit run --all-files --verbose
+        echo "debug:"
+        ls ${GITHUB_WORKSPACE}/triton_shared
 
     - name: Build/Install Triton
+      working-directory: triton_shared/triton/python
       run: |
-        export TRITON_CODEGEN_TRITON_SHARED=1
-        cd triton/python
         python3 -m pip install --upgrade pip
-        python3 -m pip install cmake==3.24
-        python3 -m pip install ninja
-        python3 -m pip uninstall -y triton
-        python3 setup.py build
-        python3 -m pip install --no-build-isolation -vvv '.[tests]'
+        python3 -m pip install cmake==3.24 ninja pytest-xdist
+        sudo apt-get update -y
+        sudo apt-get install -y ccache clang lld
+        export TRITON_PLUGIN_DIRS="${GITHUB_WORKSPACE}/triton_shared"
```

Comment:
Is there a reason you export this variable but put the others on the python build command?

Reply:
I copied the build command from triton main, and since the line is already long, I just put this in another line instead. Also this variable is special so I want others to easily see it.